client/src/app/Main.res

```diff
@@ -213,7 +214,7 @@ let processFocus = (m: model, focus: AppTypes.Focus.t): modification =>
       })
     | (_, _) =>
       switch page {
-      | SettingsModal(tab) => Many(SettingsView.getModifications(m, OpenSettingsView(tab)))
+      // | SettingsModal(tab) => Many(SettingsUpdate.getModifications(m, Open(tab)))
       | _ => NoChange
```

Comment:
I think this can be simplified to just `NoChange`?

Reply:
Ah, actually - this is likely relevant to directing to the modal when the page opens. If so, ignore.

Reply:
I pushed some new commits to fix this. This is where the settings modal is loaded if you go to a `/settings/` url.